commit 333dc6041f339f9dc7ef9b8954a069b49f292d03
Author: q66 <q66@chimera-linux.org>
Date:   Sun Nov 13 06:29:31 2022 +0100

    remove bad version constraint
    
    This version scheme is not accepted by our dep scanner and we
    never really had an older version anyway.

diff --git a/meson.build b/meson.build
index af4a6bc..5314479 100644
--- a/meson.build
+++ b/meson.build
@@ -874,7 +874,7 @@ caca_opt = get_option('caca').require(
     get_option('gpl'),
     error_message: 'the build is not GPL!',
 )
-caca = dependency('caca', version: '>= 0.99.beta18', required: caca_opt)
+caca = dependency('caca', required: caca_opt)
 features += {'caca': caca.found()}
 if features['caca']
     dependencies += caca
